### PR TITLE
chore(release): publish packages

### PR DESCRIPTION
**Pull request by naipaka**
- altive_lints@1.11.0

## 🔗 Related Issues

- None

## 🙌 What's Done

- `melos version`

## ✍️ What's Not Done

- None

## 🖼️ Image Differences

| Before    | After     |
| --------- | --------- |
| __image__ | __image__ |

## 🤼 Desired Review Method

- [ ] Correction Commit
- [ ] Pair programming

> [!NOTE]
> It is possible that a reviewer's will may cause a method to be implemented that is not selected.

## Pre-launch Checklist
- [x] I have reviewed my own code.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I updated/added relevant documentation (doc comments with ///).

### CODE REVIEW

**Review by k-nkmr — APPROVED** [review]
LGTM

File: packages/altive_lints/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+## 1.11.0
+
```
Comment (riscait) [inline packages/altive_lints/CHANGELOG.md:2]: badge
How about adding a heading about Custom Lint?

```suggestion

### Add Custom Lint

If you want to adopt it, specify custom_lints in analysis_options.yaml.
Detailed notation is here: [altive_lints/example/analysis_options.yaml](https://github.com/altive/altive_lints/blob/main/packages/altive_lints/example/analysis_options.yaml)

```

**Review by riscait — APPROVED** [review]
LGTM
A great set of rules!

File: packages/altive_lints/CHANGELOG.md (hunk 1 above)
Comment (naipaka) [reply packages/altive_lints/CHANGELOG.md:2]: You're right, it would be better to include that information! I'll commit the changes! 🙏

**Pull request merged**